SConstruct

```diff
@@ -30,6 +26,10 @@ env.Decider('MD5-timestamp') # Only computes hash if time-stamp changed
 env.EXTENSIONS = ['.eps', '.pdf', '.lyx'] # Extensions to be used when scanning for source files in BuildLyx.
 SourceFileScanner.add_scanner('.lyx', Scanner(gslab_scons.misc.lyx_scan, recursive = True))
 
+# Load paths
+env['PATHS'] = yaml.load(open("constants.yaml", 'rU'))
```

Comment:
I don't think this in general is standard scons practice, so I wouldn't expect there to be an 'accepted scons-specific paths variable name'.